application.py

```diff
@@ -1,158 +1,7 @@
-import os
```

Comment:
I think this branch is based of another branch and not master? merging in master should make a lot of these "changes" go away.